src/Microsoft.IdentityModel.JsonWebTokens/JwtHeaderParameterNames.cs

```diff
@@ -28,69 +28,69 @@
 namespace Microsoft.IdentityModel.JsonWebTokens
 {
     /// <summary>
-    /// List of header parameter names see: http://tools.ietf.org/html/rfc7519#section-5.
+    /// List of header parameter names see: https://datatracker.ietf.org/doc/html/rfc7519#section-5.
     /// </summary>
     public struct JwtHeaderParameterNames
     {
         /// <summary>
-        /// see:https://tools.ietf.org/html/rfc7515#section-4.1.1
+        /// See: https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.1
         /// </summary>
         public const string Alg = "alg";
 
         /// <summary>
-        /// see:https://tools.ietf.org/html/rfc7515#section-4.1.10
-        /// also:https://tools.ietf.org/html/rfc7519#section-5.2
+        /// See: https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.10
+        /// also:https://datatracker.ietf.org/doc/html/rfc7519#section-5.2
         /// </summary>
         public const string Cty = "cty";
 
         /// <summary>
-        /// see:https://tools.ietf.org/html/rfc7516#section-4.1.2
+        /// See: https://datatracker.ietf.org/doc/html/rfc7516#section-4.1.2
         /// </summary>
         public const string Enc = "enc";
 
         /// <summary>
-        /// see:https://tools.ietf.org/html/rfc7518#section-4.7.1.1
+        /// See: https://datatracker.ietf.org/doc/html/rfc7518#section-4.7.1.1
         /// </summary>
         public const string IV = "iv";
 
         /// <summary>
-        /// see:https://tools.ietf.org/html/rfc7515#section-4.1.2
+        /// see:https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.2
```

Comment:
```suggestion
        /// See: https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.2
```

Reply:
Thanks. Missed that one, good catch. Fixed in e89d4fbafb9ac32deb2b8b8a704e473b9e647669